### DISCUSSION TITLE
First monthly gem5 developers meeting today! Oct 12th '23

### DISCUSSION THREAD

**Discussion by powerjg** [gem5/gem5#435] [Announcements]
We will be hosting the first monthly gem5 developer's meeting October 12 on zoom at **9am PT**.

[Join via this link](https://ucdavis.zoom.us/j/98240233429?pwd=L0J6US9seS9qSzd4ek9MaldOZm5UUT09)

The agenda is shown below. We invite everyone in the gem5 community to join. The discussion will be focused on *gem5 development*. Specifically, we will discuss ongoing development projects, synchronize on shared development projects, and discuss the short-term development goals.

If you would like to add anything to the agenda please create a comment below or reach out to Jason (@powerjg; jason@lowepower.com). 

## Agenda

- Introductions
- Current project status
- Updates on github workflow
  - Overview of how to use github
  - Things that are going really well:
  - Things that could be going better
- gem5 development discussions
  - Recent Arm improvements
  - Recent AMD GPU improvements
  - Recent RISC-V RVV improvements
  - Incoming changes for fetch-directed prefetching
    - https://github.com/gem5/gem5/pull/355
    - https://github.com/gem5/gem5/pull/359 
  - Incoming changes to gem5's build system (kconfig)
    - https://github.com/gem5/gem5/pull/69
- (if time) Community discussions
  - Pre-commit updates
  - clang-format
  - removing change-id
  - removing tags
  - merge queue
- Items from the floor

Meeting slides:
[2023-10-12 monthly meeting.pdf](https://github.com/gem5/gem5/files/12887250/2023-10-12.monthly.meeting.pdf)

**Comment by BobbyRBruce** [comment]
A recording of this meeting can be round here: https://youtu.be/bfpeo5-5ijI?si=l9ua3sf6RRdjLeT6

Here is a copy of the slides:

* [slides.pptx](https://github.com/gem5/gem5/files/13369084/slides.pptx)
* [slides.pdf](https://github.com/gem5/gem5/files/13369086/slides.pdf)
